crowbar_framework/app/assets/stylesheets/content/panel.scss

```diff
@@ -0,0 +1,55 @@
+.panel-heading {
+  border-bottom: 0;
+}
+
+.panel .panel-heading h3 {
+  padding: 0;
+  margin: 0;
+}
+.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
+  border: 1px;
+  border-left: 7px;
+  border-color: #ddd;
+  border-style: solid;
+  border-radius: 5px;
+  margin: 5px;
+  box-shadow: 1px 1px 0px #eee;
+}
+
+.panel-heading+.list-group .list-group-item:first-child {
+  border-top-width: 1px;
+}
+
+.panel>.list-group:last-child .list-group-item:last-child, .panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{
+  border-bottom: 1px solid #ccc; ;
+}
+
+.status-border-unready {
+  border-left-color: #ddd !important;
+  animation-name: blink;
+  animation-duration: 1400ms;
+  animation-iteration-count: infinite;
+}
+
+
+.status-border-ready {
+  border-left-color: #599517 !important;
+}
+
+.status-border-failed {
+  border-left-color: #FF001F !important;
+}
+
+.status-border-unknown {
+  border-left-color: #ddd !important;
+}
+
+.status-border-upgrade {
+  border-left-color: #4A90E2 !important;
+}
+
+@keyframes blink {
+  0% {border-left-color: #ddd;}
+  50% {border-left-color: #F8E71C;}
```

Comment:
Color literals like `#F8E71C` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F8E71C` should be written as `#f8e71c`